File: 1383/1383.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var maximumPerformanceOfATeam = function(root) {
    if (!root) return 0;
    return 1 + Math.max(maximumPerformanceOfATeam(root.left), maximumPerformanceOfATeam(root.right));
};

File: 1383/1383.go
func maximumPerformanceOfATeam(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := maximumPerformanceOfATeam(root.Left)
	r := maximumPerformanceOfATeam(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}